### DISCUSSION TITLE
Images resize/fit the card automatically when adding a cover

### DISCUSSION THREAD

**Discussion by brettcurtis** [GitbookIO/community#255] [0. General bugs and questions] [answered]
Be slick if, when adding a cover to an image, Gitbook would automatically fit them to the card.  What size images are the cards looking for to fit currently? 

Here is an example page: https://docs.osinfra.io/product-guides/google-cloud-platform/landing-zone#platform-dependencies - Ideally, those images would fit and center.

**Comment by addisonschultz** [comment, marked as answer]
Thanks for raising this! I think we can highlight the correct sizes a bit more - for reference, you can find the sizes in our documentation! 

https://docs.gitbook.com/content-creation/blocks/cards#how-to-create-a-card

Here are the ratios that work best: 

16:9 (eg. 1920px x 1080px)
4:3 (eg. 1024X768)
1:1 (eg. 500px x 500px)

**Reply by brettcurtis** [reply]
Bah, sorry I didn't dig in deep enough. Thanks for the link!

**Reply by addisonschultz** [reply]
No problem 😄
